packages/create-plugin/templates/app/src/pages/PageFour.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { GrafanaTheme2, PageLayoutType } from '@grafana/data';
+import { LinkButton, useStyles2 } from '@grafana/ui';
+import { ROUTES } from '../constants';
+import { prefixRoute } from '../utils/utils.routing';
+import { testIds } from '../components/testIds';
+import { PluginPage } from '@grafana/runtime';
+
+export function PageFour() {
+  const s = useStyles2(getStyles);
+
+  return (
+    <PluginPage layout={PageLayoutType.Canvas}>
+      <div className={s.page} data-testid={testIds.pageFour.container}>
+        <div className={s.container}>
+          <LinkButton data-testid={testIds.pageFour.navigateBack} icon="arrow-left" href={prefixRoute(ROUTES.One)}>
+            Back
+          </LinkButton>
+          <div className={s.content}>This is a full-width page without a navigation bar.</div>
+        </div>
+      </div>
+    </PluginPage>
```

Comment:
In case we would like this to be a full-width page I think we can just remove the `<PluginPage>` wrapper? 🤔

Reply:
Silly me, it is using the `canvas` layout. Let me resolve this comment.